packages/search/package.json

```diff
@@ -0,0 +1,52 @@
+{
+	"name": "@automattic/search",
+	"version": "1.0.0-alpha.3",
```

Comment:
Nit: we haven't released that yet, should it be `1.0.0-alpha` or `beta` or even just `1.0.0`?